src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/smoke/person/SentencesSmokeTest.kt

```diff
@@ -36,11 +36,27 @@ class SentencesSmokeTest : DescribeSpec(
           "data": [
             {
               "dateOfSentencing": "2019-08-24",
-              "isActive": null
+              "isActive": null,
+              "terms": [
+                {
+                  "years": 1,
+                  "months": 2,
+                  "weeks": 3,
+                  "days": 4
```

Comment:
I don't understand this smoke test, why would we have 'null' in the isActive part yet data in the terms?